parlai/tasks/wrapper/agents.py

```diff
@@ -40,7 +41,13 @@ def add_cmdline_args(cls, parser):
             help='The task whose fields will be manipulated.',
         )
         known_args, _ = parser.parse_known_args(nohelp=True)
-        parser.add_task_args(known_args.wrapper_task)
+        try:
+            parser.add_task_args(known_args.wrapper_task)
+        except RuntimeError:
+            warn_once(
```

Comment:
why don't we want this to raise an error?

Reply:
Because there is still a use case for running this code without adding task args, specifically when using a wrapper teacher as part of a unit test in which args have to be passed in as an opt into `testing_utils.eval_model()` as opposed to using ParlaiParser